langgraph/utils.py

```diff
@@ -148,6 +148,31 @@ def extend(
         )
 
 
+def _isasyncgencheck(thing: RunnableLike) -> bool:
+    return (
+        inspect.isasyncgenfunction(thing)
+        or inspect.isasyncgen(thing)
+        or hasattr(thing, "__call__")
```

Comment:
This seems recursive, which I don't think is on purpose?

Reply:
tired brain. ya